packages/app-desktop/gui/NoteEditor/utils/useFormNote.ts

```diff
@@ -219,6 +231,7 @@ export default function useFormNote(dependencies: HookDependencies) {
 
 		return () => {
 			cancelled = true;
+			autoFocusCancelled = true;
```

Comment:
That looks fine but can't we rely only on the `cancelled` variable? It looks like your variable `autoFocusCancelled` will always have the same value as `cancelled`, which means it's not needed

Reply:
You have a good point; I was thinking about that too but I wasn't sure if changing `cancelled` would break something else so I was a bit hesitant on that. I'll apply the change